packages/reports/addon/components/navi-tag-input.js

```diff
@@ -29,18 +30,80 @@ export default TagInput.extend({
    */
   tagComponent: 'navi-tag-input/tag',
 
+  /**
+   * @property {Boolean} splitOnPaste - whether or not to listen for paste input and handle importing lists
+   */
+  splitOnPaste: false,
+
   /**
    * Ignore comma so that only 'enter' key adds new tags
    *
    * @param {InputEvent} e - The keydown input event
    * @override https://github.com/calvinlough/ember-tag-input/blob/v1.2.2/addon/components/tag-input.js#L74-L106
    */
   _onInputKeyDown(e) {
-    if (e.which === 118 || e.target.value === '') {
+    if (e.which === 188) {
```

Comment:
Could make a const for 188 called `COMMA_KEY_CODE` instead of using a comment below this